src/deployment/deployment-hapi-plugin.ts

```diff
@@ -146,20 +144,19 @@ class DeploymentHapiPlugin {
     }, {
       method: 'GET',
       path: '/raw-deployment-handler/{param*}',
-      handler: this.directoryHandler('key'),
+      handler: this.directoryHandler(PREKEY),
       config: {
         auth: 'customAuthorize',
```

Comment:
Also: what's `customAuthorize`? Did I understand it correctly that it makes the user authorized as long as it's a valid sub claim?